wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -12,8 +12,11 @@ def get_indicator_data(profile_key_metric, geographies):
 
 
 def absolute_value(profile_key_metric, geography):
-    data = get_indicator_data(profile_key_metric, [geography]).first().data
-    return MetricCalculator.absolute_value(data, profile_key_metric, geography)
+    indicator_data = get_indicator_data(profile_key_metric, [geography])
+    if indicator_data.count() > 0:
```

Comment:
no we have some duplication. can we extract that into a method and call if from both places?